Cabal/Distribution/Compat/ResponseFile.hs

```diff
@@ -57,8 +56,6 @@ expandResponse = fmap concat . mapM expand
     expand x = return [x]
 
     readFileExc f =
-      readFile f `catch` \(e :: IOException) -> do
-        hPutStrLn stderr $ "Error while expanding response file: " ++ show e
-        exitFailure
-
-#endif
+      readFile f `catch` \(_e :: IOException) -> do
+        hPutStrLn stderr $ "Response file `@" ++ f ++ "` does not exist, assuming literal argument."

```

Comment:
do we really have to be that noisy?

Reply:
Hmm yeah, we can't use `notice` et al. here though since we don't know the verbosity level yet. Let's just be silent.

Reply:
For reference, here's GNU Binutils' implementation of `@file`-expansion:

http://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;a=blob;f=libiberty/argv.c;h=6444896f995a8e4abfb6a1edc510f3a4bbe1a30b;hb=HEAD#l342